File: ListView/Custom ListView/MainActivity.java
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ListView;
import android.widget.Toast;
public class MainActivity extends AppCompatActivity {
    private ListView listView;
    private String [] countryNames;
    private int[] flags ={
            R.drawable.ic_launcher_background,R.drawable.ic_launcher_background,
            R.drawable.ic_launcher_background,R.drawable.ic_launcher_background,
            R.drawable.ic_launcher_background
    };
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        listView = findViewById(R.id.activity_main_ListView);

        countryNames = getResources().getStringArray(R.array.country_names);

        CustomAdapter adapter = new CustomAdapter(this, countryNames, flags);

        listView.setAdapter(adapter);

        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                String value = countryNames[position];
                Toast.makeText(MainActivity.this, value+" Selected", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: CheckBox/Kotlin/MainActivity.kt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.RadioButton
import android.widget.Toast

import com.example.kotlinviewbinding.databinding.ActivityMainBinding
import java.lang.StringBuilder

class MainActivity : AppCompatActivity() {

    private  lateinit var  binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnDisplay.setOnClickListener {
            var totalAmount: Int = 0
            var result = StringBuilder()

            result.append("Seleted Items")

            if (binding.checkBox1.isChecked) {
                result.append("\nPizza 100$")
                totalAmount += 100
            }

            if (binding.checkBox2.isChecked) {
                result.append("\nCoffee 50$")
                totalAmount += 50
            }

            if (binding.checkBox3.isChecked) {
                result.append("\nPizza 120$")
                totalAmount += 120
            }

            result.append("\nIn total: " + totalAmount + "$")
            Toast.makeText(applicationContext, result.toString(), Toast.LENGTH_SHORT).show()
        }
    }
}
